### PR TITLE
added CaptureUserData config toggle

### PR DESCRIPTION
**Pull request by dil-cgrealy**

### CODE REVIEW

File: src/Elastic.Apm/Api/User.cs [hunk 1]
```diff
@@ -17,6 +17,7 @@ public class User
 		public string UserName { get; set; }
 
 		public override string ToString() =>
-			new ToStringBuilder(nameof(User)) { { "Id", Id }, { "UserName", UserName }, { "Email", Email } }.ToString();
+			new ToStringBuilder(nameof(User)) { { "Id",
+					Id }, { "UserName", UserName }, { "Email", Email } }.ToString();
```
Comment (ange-d) [inline src/Elastic.Apm/Api/User.cs]: Looks like just a whitespace change - do we want this?

Comment (dil-cgrealy) [reply src/Elastic.Apm/Api/User.cs]: Probably not. I'll remove it

File: test/Elastic.Apm.AspNetCore.Tests/TestConfigs/appsettings_valid.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
     "ServiceNodeName": "Instance1",
     "ServiceVersion": "2.1.0.5",
     "Environment": "staging",
+    "CaptureUserData":  "false",
```
Comment (dbrougham) [inline test/Elastic.Apm.AspNetCore.Tests/TestConfigs/appsettings_valid.json]: Extra white space

**Review by ange-d — APPROVED** [review]
I am assuming you have run all tests according to contribution guidelines locally.

**Review by dbrougham — APPROVED** [review]

**Comment by dil-cgrealy** [comment]
> I am assuming you have run all tests according to contribution guidelines locally.

yes. All tests are passing as per contribution guidelines

**Comment by cgrealy** [comment]
> I am assuming you have run all tests according to contribution guidelines locally.

yes. All tests are passing as per contribution guidelines

**Pull request closed without merging**